Jarvis-Platform/jarvis-studio
# Feature/components structure + layouts

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Reorganize arborescence of components & remove unused ones in `src/components` + create layouts


## Todos
- [x] Remove `src/components/tmp` folder & files inside (update imports in `.vue` files to new components).
- [x] Move file in `src/components/app/layouts` to `src/components/app`
- [x] Organize `src/components/data-workflows`


## Deploy Notes
Review & Merge on `master`


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/components-structure
```

1. Run `npm run serve` => No import error should appear
2. Take a lot to components / layouts structure


## Impacted Areas in Application
List general components of the application that this PR will affect:

- `src/components`
- `src/layouts`
